packages/flutter/lib/src/widgets/framework.dart

```diff
@@ -1545,7 +1546,7 @@ abstract class ProxyWidget extends Widget {
 abstract class ParentDataWidget<T extends ParentData> extends ProxyWidget {
   /// Abstract const constructor. This constructor enables subclasses to provide
   /// const constructors so that they can be used in const expressions.
-  const ParentDataWidget({ Key key, Widget child })
+  const ParentDataWidget({ Key? key, Widget? child })
```

Comment:
child should be non-nullable and required

Reply:
Some null may be here accros all children.

Reply:
Done